ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -181,7 +180,7 @@ objects:
       - name: AWS_SHARED_CREDENTIALS_FILE
         value: /etc/openshift-installer/.awscred
       - name: KUBECONFIG
-        value: /tmp/cluster/generated/auth/kubeconfig
+        value: /tmp/artifacts/installer/auth/kubeconfig
```

Comment:
Putting kubeconfig in artifacts isn't a great idea if we accidentally put a secret in there.

Reply:
(i.e. if something shows up in there that is longer lived than the cluster)

Reply:
> Putting kubeconfig in artifacts isn't a great idea if we accidentally put a secret in there.

I'd guessed that we were we generating the secret for this CI run on the fly.  Is that not accurate?  Or is this just a "watch out for long-lived secrets in the future"?